src/code/z_map_disp.c

```diff
@@ -1,109 +1,1881 @@
 #include "global.h"
+#include "gfx.h"
+#include "assets/interface/icon_item_dungeon_static/icon_item_dungeon_static.h"
+#include "assets/interface/parameter_static/parameter_static.h"
+#include "overlays/actors/ovl_En_Door/z_en_door.h"
+#include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_disp/func_80102E40.s")
+void MapDisp_DestroyMapI(PlayState* play);
+void MapDisp_InitMapI(PlayState* play);
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_disp/func_80102E90.s")
+static s32 sLoadTextureBlock_siz[4] = {
+    G_IM_SIZ_4b,
+    G_IM_SIZ_8b,
+    G_IM_SIZ_16b,
+    G_IM_SIZ_32b,
+};
+static s32 sLoadTextureBlock_siz_LOAD_BLOCK[4] = {
+    G_IM_SIZ_4b_LOAD_BLOCK,
+    G_IM_SIZ_8b_LOAD_BLOCK,
+    G_IM_SIZ_16b_LOAD_BLOCK,
+    G_IM_SIZ_32b_LOAD_BLOCK,
+};
+static u32 sLoadTextureBlock_siz_INCR[4] = {
+    G_IM_SIZ_4b_INCR,
+    G_IM_SIZ_8b_INCR,
+    G_IM_SIZ_16b_INCR,
+    G_IM_SIZ_32b_INCR,
+};
+static s32 sLoadTextureBlock_siz_SHIFT[4] = {
+    G_IM_SIZ_4b_SHIFT,
+    G_IM_SIZ_8b_SHIFT,
+    G_IM_SIZ_16b_SHIFT,
+    G_IM_SIZ_32b_SHIFT,
+};
+static u32 sLoadTextureBlock_siz_BYTES[4] = {
+    G_IM_SIZ_4b_BYTES,
+    G_IM_SIZ_8b_BYTES,
+    G_IM_SIZ_16b_BYTES,
+    G_IM_SIZ_32b_BYTES,
+};
+static u32 sLoadTextureBlock_siz_LINE_BYTES[4] = {
+    G_IM_SIZ_4b_LINE_BYTES,
+    G_IM_SIZ_8b_LINE_BYTES,
+    G_IM_SIZ_16b_LINE_BYTES,
+    G_IM_SIZ_32b_LINE_BYTES,
+};
```

Comment:
Bump on this. Please use the names from OoT.

Reply:
idk, personally I prefer to deviate from the naming convention because it mirrors how constants are normally composed within the original macros (e.g. sLoadTextureBlock_siz_BYTES replaces `siz##_BYTES` in the original gDPLoadTextureBlock).

If you're still against this idea, then I still don't like copying the OoT names because they don't quite make sense. `sLoadBlockImageSizes` to me reads as if this is an array of ImageSizes of LoadBlock? which doesn't make sense.

Reply:
Hmm, with that in mind the names don't seem too bad. I think I still prefer something more along our style lines, but would like more opinions.